overlord/configstate/configcore/runwithstate.go

```diff
@@ -138,6 +138,8 @@ func applyHandlers(cfg config.Conf, handlers []configHandler) error {
 	return nil
 }
 
+// Early applies the specified configuration earlier than it would
+// normally be applied, using
```

Comment:
this isn't finished...  also it's more about applying the configuration that can be applied early (before snaps are seeded) than any/all configuration earlier

Reply:
I just ended up dropping this